server/aws-lsp-codewhisperer/src/language-server/chat/telemetry/chatTelemetryController.ts

```diff
@@ -188,10 +188,11 @@ export class ChatTelemetryController {
         this.#telemetry.emitMetric({
             name: ChatTelemetryEventName.InteractWithAgenticChat,
             data: {
-                [CONVERSATION_ID_METRIC_KEY]: this.getConversationId(tabId),
+                [CONVERSATION_ID_METRIC_KEY]: this.getConversationId(tabId) ?? '',
```

Comment:
non-blocker: not sure what relies on this, but could be worth passing an explicit 'unknown-tab' value here.